<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>111111</width>
    <height>111111</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NovelTools</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>571</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>小说下载</string>
     </attribute>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>15</y>
        <width>20</width>
        <height>165</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>50</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>下载地址</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="save_novel_path_by_page">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>75</y>
        <width>170</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="select_download_type">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>shubao12.com</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>75</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存目录</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_download_url">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>190</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>50</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>下载模式</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_download_start_executr">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>130</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QComboBox" name="select_download_mode">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>80</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>单章下载</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>批量下载</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>自动格式化</string>
     </attribute>
     <widget class="QPushButton" name="button_format_start_execute">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>15</y>
        <width>20</width>
        <height>165</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="button_format_orgin_file_path">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>源文件目录</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>50</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>换行模式</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_format_result_file_path">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>保存目录</string>
      </property>
     </widget>
     <widget class="QComboBox" name="select_format_type">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>88</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>全局模式A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>全局模式B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>单行模式</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="line_format_result_file_path">
      <property name="geometry">
       <rect>
        <x>94</x>
        <y>90</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_format_orgin_file_path">
      <property name="geometry">
       <rect>
        <x>94</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>手动格式化</string>
     </attribute>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>15</y>
        <width>20</width>
        <height>165</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>23</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>格式处理</string>
      </property>
     </widget>
     <widget class="QComboBox" name="manual_comboBox_select_format_mode">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>换行校验</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>去除广告</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>词语纠错</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>繁转简</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>字符替换</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="manual_input_string_select_mode">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>115</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="manual_select_string_mode">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>字符查询</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>正则查询</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="manual_input_string_result_mode">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>195</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="manual_button_select">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>70</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manual_button_select_next_one">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>150</y>
        <width>70</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>查找下一个</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manual_button_execute">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>70</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manual_button_get_file_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>获取文本</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manual_button_save_file">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>保存文本</string>
      </property>
     </widget>
     <widget class="QListWidget" name="manual_file_item_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>56</y>
        <width>261</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="manual_button_other_save_file">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>另存为</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manual_button_replace">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>150</y>
        <width>70</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>替换</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>73</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="manual_button_callback">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>70</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>撤回</string>
      </property>
     </widget>
     <widget class="QComboBox" name="manual_comboBox_select_change_line_mode">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>全局换行</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>单列换行</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>23</y>
        <width>30</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>换行</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>591</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>日志信息</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>571</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
